src/semgrep_agent/meta.py

```diff
@@ -43,6 +44,11 @@ def repo(self) -> gitpython.Repo:  # type: ignore
     def repo_name(self) -> str:
         return Path.cwd().name
 
+    @cachedproperty
+    def repo_remote_origin(self) -> str:
+        remotes: List[str] = [remote.url for remote in self.repo.remotes]
```

Comment:
```suggestion
        remotes: List[Any] = sorted(self.repo.remotes, key=lambda x: x.remote == "origin")
        return remotes[-1].url if remotes else None
```

1. Other methods return `None` when no answer is available
2. This prioritizes `origin` if available